plugins/rikki/loungeviews/components/participationoverview/default.htm

```diff
@@ -30,10 +30,20 @@ <h1>{{__SELF__.season.title}} Participation</h1>
                 class="btn btn-sm btn-secondary center"
                 data-request-data="season_id: {{__SELF__.season.id}}"
                 data-request="{{ __SELF__ }}::onSlothSignup">
-            Sign up as a free agent!
+            Sign up as a free agent
         </button>
         <br><br>
     </div>
+    {% elseif __SELF__.user and __SELF__.signedUp and __SELF__.user.sloth.region_id == __SELF__.season.region_id %}
+    <div>
+        No longer want to particpate as a free agent?<br>
```

Comment:
Is there something like 'You are signed up as a free agent' on this page? If not, please add it before this.